sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -756,7 +756,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief The maximum number of threads that may be used in a parallel transfer.
```

Comment:
General doc feedback for all these option types: Should we document the default values in the doc comments?